cli.js

```diff
@@ -121,6 +121,12 @@ program
 
 							writeStream.write((wroteData ? '\n' : '') + data.join('\n'));
 							break;
+
+						case 'stdout':
```

Comment:
I think this switch is about the file output format, not the output format in general. I guess the `outputFormat` variable should be renamed to be more clear.